website/content/docs/upgrading/upgrade-to-1.13.x.mdx

```diff
@@ -90,6 +92,20 @@ forwarded to the active node of this cluster.
 
 As a workaround, submit revocation requests to the active node only.
 
+### Slow Startup Time When Storing PKI Certificates
```

Comment:
Why is this part of this PR?

Reply:
🤔 Oh it was a merge conflict from Ent but I seem to have put this back in the wrong place. Let me fix that.

Reply:
nvm, missed that it was just moved.